src/Nuvem.PharmacyManagementSystem.Pharmacy.Services/Models/MappingHelper.cs

```diff
@@ -0,0 +1,23 @@
+namespace Nuvem.PharmacyManagementSystem.Pharmacy.Services.Models;
+
+public static class MappingHelper<T1, T2>
+{
+    public static T1 Map(T2 source)
+    {
+        T1 targetItem = Activator.CreateInstance<T1>();
+        var props = typeof(T2).GetProperties();
+        var targetProps = typeof(T1).GetProperties();
```

Comment:
Please verify - if GetProperties() invokes reflection...

Reply:
Yes, it does. I debugged through it while writing the code.